shell/server/notifications-server.js

```diff
@@ -0,0 +1,128 @@
+// Sandstorm - Personal Cloud Sandbox
+// Copyright (c) 2016 Sandstorm Development Group, Inc. and contributors
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const Capnp = Npm.require("capnp");
+const SupervisorCapnp = Capnp.importSystem("sandstorm/supervisor.capnp");
+const SystemPersistent = SupervisorCapnp.SystemPersistent;
+
+logActivity = function (grainId, identityId, event) {
```

Comment:
This appears to be another global.  It would be better if it lived inside a file under `shell/imports/server/` and were explicitly imported in the places it is used.